evalai/challenges.py

```diff
@@ -11,66 +11,43 @@
 
 @click.group(invoke_without_command=True)
 @click.pass_context
-def challenges(ctx):
-    """
-    Challenges and related options.
-    """
-    if ctx.invoked_subcommand is None:
-        welcome_text = ("Welcome to the EvalAI CLI. Use evalai"
-                        "challenges --help for viewing all the options.")
-        echo(welcome_text)
-
-
-@click.group(invoke_without_command=True, name='list')
-@click.pass_context
 @click.option('--participant', is_flag=True,
               help="Show the challenges that you've participated")
 @click.option('--host', is_flag=True,
               help="Show the challenges that you've hosted")
-def list_challenges(ctx, participant, host):
+def challenges(ctx, participant, host):
     """
     Used to list challenges.
-    Invoked by running `evalai challenges list`
+    Invoked by running `evalai challenges`
     """
-    if participant:
-        get_challenge_count(host, participant)
-    elif host:
+    if participant or host:
         get_challenge_count(host, participant)
```

Comment:
I'm not sure the naming of this function makes sense. It makes it seem like it is returning a number.

Reply:
@isht3 any updates on this? Else everything else looks good.

Reply:
No, not on this PR anymore. I've made an issue which I would go back to after the primary features are done.

Reply:
@isht3 I think we should fix this here only. Can you please add it quickly?